<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BitTreeMainWindow</class>
 <widget class="QMainWindow" name="BitTreeMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BitTreeMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionclear"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>遍历</string>
    </property>
    <addaction name="actionpre"/>
    <addaction name="actionmid"/>
    <addaction name="actionpro"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>线索化</string>
    </property>
    <addaction name="actionpre_2"/>
    <addaction name="actionmid_2"/>
    <addaction name="actionpro_2"/>
    <addaction name="actionclear_2"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>叶子结点计数</string>
    </property>
    <addaction name="actioncal"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionclear">
   <property name="text">
    <string>清空二叉树</string>
   </property>
  </action>
  <action name="actionpre">
   <property name="text">
    <string>先序遍历</string>
   </property>
  </action>
  <action name="actionmid">
   <property name="text">
    <string>中序遍历</string>
   </property>
  </action>
  <action name="actionpro">
   <property name="text">
    <string>后序遍历</string>
   </property>
  </action>
  <action name="actionpre_2">
   <property name="text">
    <string>先序线索化</string>
   </property>
  </action>
  <action name="actionmid_2">
   <property name="text">
    <string>中序线索化</string>
   </property>
  </action>
  <action name="actionpro_2">
   <property name="text">
    <string>后序线索化</string>
   </property>
  </action>
  <action name="actionclear_2">
   <property name="text">
    <string>取消线索化</string>
   </property>
  </action>
  <action name="actioncal">
   <property name="text">
    <string>叶子结点计数</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
